lib/TransformLambdaSpecialization.hpp

```diff
@@ -55,12 +56,25 @@ static inline Constant *getConstant(LLVMContext &Ctx, Type *ArgType,
 class TransformLambdaSpecialization {
 public:
   static void transform(Module &M, Function &F,
-                        SmallVector<RuntimeConstant, 8> &RC) {
+                        DenseMap<StringRef, SmallVector<RuntimeConstant, 4>> &RCMap) {
     auto LambdaClass = F.getArg(0);
     PROTEUS_DBG(Logger::logs("proteus")
                 << "TransformLambdaSpecialization::transform" << "\n");
     PROTEUS_DBG(Logger::logs("proteus") << "\t args" << "\n");
-    for (auto &Arg : RC) {
+
+    const std::string Operator{llvm::demangle(F.getName())};
+    std::size_t Sep = Operator.find("::");
+    std::size_t SecondSep = Operator.find("::", Sep+1);
+
+    const std::string Symbol = Operator.substr(0, SecondSep);
```

Comment:
optional: use `slice` on the `StringRef` of the function name to isolate the `Symbol`